core/templates/components/forms/custom-forms-directives/thumbnail-uploader.directive.html

```diff
@@ -12,26 +12,28 @@
       <oppia-thumbnail-display ng-show="!thumbnailIsLoading"
                                [img-src]="editableThumbnailDataUrl"
                                [classes]="['img-thumbnail', 'protractor-test-custom-photo']"
-                               [background]="getBgColor()">
+                               [background]="getBgColor()"
+                               [aspectRatio]="4:3">
```

Comment:
Should we use the aspectRatio that is passed to "thumbnail-uploader.directive" instead of hardcoding it here?

Reply:
Yes, you are right.
Done, Thanks